frontend/src/components/Register.jsx

```diff
@@ -0,0 +1,97 @@
+import { createUserWithEmailAndPassword } from "firebase/auth";
+import React, { useState } from "react";
+import { auth, db } from "../config/firebase-config";
+import { setDoc, doc } from "firebase/firestore";
+import { toast } from "react-toastify";
+
+function Register() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [fname, setFname] = useState("");
+  const [lname, setLname] = useState("");
+
+  const handleRegister = async (e) => {
+    e.preventDefault();
+    try {
+      await createUserWithEmailAndPassword(auth, email, password);
+      const user = auth.currentUser;
+      console.log(user);
+      if (user) {
+        await setDoc(doc(db, "Users", user.uid), {
+          email: user.email,
+          firstName: fname,
+          lastName: lname,
+          photo:""
+        });
+      }
```

Comment:
can you explain this process? is `currentUser` the newly created user, or an existing one? then what are we doing with the `setDoc()` call?